src/main/java/uk/gov/hmcts/reform/civil/config/JacksonConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.civil.config;
+
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+@Configuration
+public class JacksonConfiguration {
```

Comment:
This is the class I want people to be aware of during review.

Its introduction defines a clear behaviour upon date serialization that will result in all date/time values to be serialized the same way.

Previously we would have had different formats based on the value of the date, as in these examples:
[2022-10-19 15.36.00.000000] -> "2022-10-19T15.36"
[2022-10-19 15.36.01.000000] -> "2022-10-19T15.36.01"
[2022-10-19 15.36.01.000001] -> "2022-10-19T15.36.01.000001"

With this change we will always have the same behaviour (for date/time, but similarly for dates, albeit with a different format):
[2022-10-19 15.36.00.000000] -> "2022-10-19T15.36.00"
[2022-10-19 15.36.01.000000] -> "2022-10-19T15.36.01"
[2022-10-19 15.36.01.000001] -> "2022-10-19T15.36.01"

The reason I'd like people to review this carefully is because although probably not breaking this change will affect the entire application.